apisix/init.lua

```diff
@@ -409,6 +409,33 @@ function _M.http_access_phase()
     api_ctx.route_id = route.value.id
     api_ctx.route_name = route.value.name
 
+    if route.value.script then
```

Comment:
Why run this if and else blocks eariler?

Reply:
Because the upstream_id in the init.lua file is processed before the plugin is executed, this will cause the upstream_id setting in the traffic-split plugin to become invalid. Therefore, adjust the plugin execution logic to before upstream_id processing.

Reply:
Got it,  just adjusted the order of plugins and upstream loading. It has no effect on other plugins.